### PR TITLE
Shape type and compound shape property fixes

### PR DESCRIPTION
**Pull request by dback2**
https://highfidelity.manuscript.com/f/cases/20735
https://highfidelity.manuscript.com/f/cases/20777
https://highfidelity.manuscript.com/f/cases/20801

Pre-merge test plan:
- Enter Interface in desktop mode into any domain with edit permissions
- Open Create and select or create any Model entity
- Verify you can change the model's Collision Shape and Compound Shape (for compound shape setting, set a URL to an OBJ or FBX file) properties as expected
- Exit and re-enter Create, return to the same model entity from above, verify the Collision Shape and Compound Shape properties are the same as before
- Select or create any Zone entity
- Verify the Shape Type and Compound Shape URL properties for Zones appear immediately under Parent Joint Index at the top
- Verify you can change the zone's Shape Type and Compound Shape URL (for use compound shape URL setting, set a URL to an OBJ or FBX file) properties as expected
- Also verify tooltips appear for Shape Type and Compound Shape URL properties as expected and start with "The model file"
- Exit and re-enter Create, return to the same zone entity from above, verify the Shape Type and Compound Shape URL properties are the same as before

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16572/HighFidelity-Beta-PR14744-e40eec8.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16572/HighFidelity-Beta-PR14744-e40eec8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16572/HighFidelity-Beta-Interface-PR14744-e40eec8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16572/HighFidelity-Beta-PR14744-e40eec8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16572/HighFidelity-Beta-Interface-PR14744-e40eec8.exe

File: scripts/system/assets/data/createAppTooltips.json [hunk 1]
```diff
@@ -126,6 +126,14 @@
     "bloom.bloomSize": {
         "tooltip": "The radius of bloom. The higher the value, the larger the bloom."
     },
+    "zoneShapeType": {
+        "tooltip": "The shape of the zone's geometry.",
+        "jsPropertyName": "shapeType"
+    },
+    "zoneCompoundShapeURL": {
+        "tooltip": "The OBJ file to use for the compound shape if Shape Type is \"Use Compound Shape URL\".",
```
Comment (thoys) [inline scripts/system/assets/data/createAppTooltips.json]: This could be an FBX too. Perhaps change it to model file, or compound model file

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16590/HighFidelity-Beta-PR14744-1cdd038.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16590/HighFidelity-Beta-Interface-PR14744-1cdd038.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16590/HighFidelity-Beta-PR14744-1cdd038.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16670/HighFidelity-Beta-PR14744-8903269.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16670/HighFidelity-Beta-PR14744-8903269.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16670/HighFidelity-Beta-Interface-PR14744-8903269.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16670/HighFidelity-Beta-PR14744-8903269.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16670/HighFidelity-Beta-Interface-PR14744-8903269.exe

**Comment by QliCknox** [comment]
Currently Testing

**Comment by QliCknox** [comment]
Pass

compound shapes persist

**Pull request merged**